Dockerfile

```diff
@@ -16,26 +16,83 @@
 
 FROM python:3.6-slim
 
-COPY . /opt/airflow/
+SHELL ["/bin/bash", "-c"]
+
+# Make sure noninteractie debian install is used
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Increase the value to force renstalling of all apt-get dependencies
+ENV FORCE_REINSTALL_APT_GET_DEPENDENCIES=1
+
+# Install core build dependencies
+RUN apt-get update \
```

Comment:
No problem with that. Those two dependency lists are defined separately in the original image so i split them as well, but there is no particular reason to do it. 

One reason why you would want to separate it and do separate installs (I often do that) is when you find that you need to add extra tool that you want to use (tree or top or whatever). When they are not used for build but you find it useful to have when you enter the container. Then it makes sense to add extra 'apt-get' at the end of the file so that you do not have to rebuild the whole image.

But this is not the case here, so I can easily combine it.